packages/react-form/README.md

```diff
@@ -55,11 +56,18 @@ import React from 'react';
 import {useField, useForm} from '@shopify/react-form';
 
 function MyComponent() {
-  const {fields, submit, submitting, dirty, reset, submitErrors} = useForm({
+  const {
+    fields: {title},
+    submit,
+    submitting,
+    dirty,
+    reset,
+    submitErrors,
```

Comment:
should a bound version of `undirty` be returned from here?

Reply:
I could add that in 👍